_code-blocks/zendesk/embed-code.js

```diff
@@ -1,45 +1,10 @@
ownLanguage?"":"hljs "+this.detectedLanguage},highlighted(){if(!this.autoDetect&&!e.getLanguage(this.language))return console.warn(`The language "${this.language}" you specified could not be found.`),this.unknownLanguage=!0,ae(this.code);let a={};return this.autoDetect?(a=e.highlightAuto(this.code),this.detectedLanguage=a.language):(a=e.highlight(this.language,this.code,this.ignoreIllegals),this.detectedLanguage=this.language),a.value},autoDetect(){return!this.language||Ln(this.autodetect)},ignoreIllegals(){return!0}},render(a){return a("pre",{},[a("code",{class:this.className,domProps:{innerHTML:this.highlighted}})])}};return{Component:t,VuePlugin:{install(a){a.component("highlightjs",t)}}}}var Pn={"after:highlightElement":({el:e,result:t,text:n})=>{let a=it(e);if(!a.length)return;let c=document.createElement("div");c.innerHTML=t.value,t.value=Bn(a,it(c),n)}};function Le(e){return e.nodeName.toLowerCase()}function it(e){let t=[];return function n(a,c){for(let i=a.firstChild;i;i=i.nextSibling)i.nodeType===3?c+=i.nodeValue.length:i.nodeType===1&&(t.push({event:"start",offset:c,node:i}),c=n(i,c),Le(i).match(/br|hr|img|input/)||t.push({event:"stop",offset:c,node:i}));return c}(e,0),t}function Bn(e,t,n){let a=0,c="",i=[];function d(){return!e.length||!t.length?e.length?e:t:e[0].offset!==t[0].offset?e[0].offset<t[0].offset?e:t:t[0].event==="start"?e:t}function r(l){function y(p){return" "+p.nodeName+'="'+ae(p.value)+'"'}c+="<"+Le(l)+[].map.call(l.attributes,y).join("")+">"}function o(l){c+="</"+Le(l)+">"}function u(l){(l.event==="start"?r:o)(l.node)}for(;e.length||t.length;){let l=d();if(c+=ae(n.substring(a,l[0].offset)),a=l[0].offset,l===e){i.reverse().forEach(o);do u(l.splice(0,1)[0]),l=d();while(l===e&&l.length&&l[0].offset===a);i.reverse().forEach(r)}else l[0].event==="start"?i.push(l[0].node):i.pop(),u(l.splice(0,1)[0])}return c+ae(n.substr(a))}var De=e=>{console.error(e)},rt=(e,...t)=>{console.log(`WARN: ${e}`,...t)},G=(e,t)=>{console.log(`Deprecated as of ${e}. ${t}`)},Be=ae,at=J,ot=Symbol("nomatch"),Un=function(e){let t=Object.create(null),n=Object.create(null),a=[],c=!0,i=/(^(<[^>]+>|\t|)+|\n)/gm,d="Could not find the language '{}', did you forget to load/include a language module?",r={disableAutodetect:!0,name:"Plain text",contains:[]},o={noHighlightRe:/^(no-?highlight)$/i,languageDetectRe:/\blang(?:uage)?-([\w-]+)\b/i,classPrefix:"hljs-",tabReplace:null,useBR:!1,languages:null,__emitter:Ze};function u(s){return o.noHighlightRe.test(s)}function l(s){let h=s.className+" ";h+=s.parentNode?s.parentNode.className:"";let S=o.languageDetectRe.exec(h);if(S){let w=F(S[1]);return w||(rt(d.replace("{}",S[1])),rt("Falling back to no-highlight mode for this block.",s)),w?S[1]:"no-highlight"}return h.split(/\s+/).find(w=>u(w)||F(w))}function y(s,h,S,w){let I="",ee="";typeof h=="object"?(I=s,S=h.ignoreIllegals,ee=h.language,w=void 0):(G("10.7.0","highlight(lang, code, ...args) has been deprecated."),G("10.7.0",`Please use highlight(code, options) instead.
```

Comment:
Same for this one.

Reply:
I just reverted the files to how they were on the main branch

Reply:
Got it, thanks